game/src/main/scala/io/aigar/game/Player.scala

```diff
@@ -7,9 +7,14 @@ class Player(val id: Int, startPosition: Vector2) {
   var cells = List(new Cell(0, startPosition))
 
   def update(deltaSeconds: Float, grid: Grid, players: List[Player]) {
-    val opponents = players.filterNot(_ == this)
-    cells.foreach { _.update(deltaSeconds, grid) }
-    cells.foreach { _.eats(opponents)}
+    if ( cells.isEmpty ) {
+      cells = List(new Cell(0, grid.randomPosition))
+    }
+    else {
+      val opponents = players.filterNot(_ == this)
+      cells.foreach { _.update(deltaSeconds, grid) }
```

Comment:
By not calling `update`, the AI would *not* pick a target for the new cell if the player is inactive. Why should it wait a frame? That's all the logic that we have in there for now, but I don't see why we should treat this case separately. It feels to me like this complicates the code (with two code paths) without a good reason that I can see.

In other words, I don't see the benefit of *not* updating it.